packages/expo-cli/src/commands/build/utils.ts

```diff
@@ -8,11 +8,16 @@ export async function checkIfSdkIsSupported(
   platform: 'android' | 'ios'
 ): Promise<void> {
   const isSupported = await Versions.canTurtleBuildSdkVersion(sdkVersion, platform);
+  const minimumSdkVersionSupported = await Versions.oldestSupportedMajorVersionAsync();
+  const majorSdkVersion = Number(sdkVersion.split('.')[0]);
+
   if (!isSupported) {
-    const storeName = platform === 'ios' ? 'Apple App Store' : 'Google Play Store';
     log.error(
       chalk.red(
-        `Unsupported SDK version: our app builders don't have support for ${sdkVersion} version yet. Submitting the app to the ${storeName} may result in an unexpected behavior.`
+        'Unsupported SDK version: our app builders ' +
+          (majorSdkVersion < minimumSdkVersionSupported
+            ? `no longer support SDK version ${majorSdkVersion}. Please upgrade to at least SDK ${minimumSdkVersionSupported}.`
```

Comment:
Hmmm, maybe we should suggest upgrading to the latest available SDK version? Rather than mentioning the oldest support version. What do you think?

Reply:
I thought that too, but we suggest upgrading one version at a time anyways so thought +1 was probably better for the user

Reply:
Makes sense. However, what do you think about making this message to say what's the latest SDK version?

Reply:
yeah, I'll add that 👍